src/pages/AppLogin.vue

```diff
@@ -74,22 +74,29 @@
 
 <script>
 import { onMounted } from "vue";
+import { useRouter } from "vue-router";
 
 import useArcanaWallet from "../use/arcanaWallet";
 import useToast from "../use/toast";
 
 export default {
   setup() {
-    const { toastError } = useToast();
-    const { requestSocialLogin } = useArcanaWallet();
+    const router = useRouter();
+    const { toastSuccess, toastError } = useToast();
+    const { requestSocialLogin, setHook } = useArcanaWallet();
 
     onMounted(async () => {
       document.title = "Login | Arcana Demo";
+      setHook("accountsChanged", (params) =>
```

Comment:
The intent here is that if the user clicks the social icon within the wallet, the parent app needs to detect a successful login and redirect to the my files page.